.github/workflows/sync-to-codecommit.yaml

```diff
@@ -27,5 +27,5 @@ jobs:
       - run: git remote add codecommit ${{ secrets.AWS_CODECOMMIT_REPO_URL }}
       - run: git checkout master
       - run: git push codecommit master
-      - run: git checkout al2023
-      - run: git push codecommit al2023
+      - run: git checkout main
```

Comment:
Probably need to revert the unintentional changes in this PR

Reply:
These came from Carter committing to this branch. Not sure what is happening there, so I will reach out to him